src/pages/en/microsoft/index.md

```diff
@@ -0,0 +1,126 @@
+</details>
+
+### Microsoft Office applications
+{:#office_apps}
+
+**Available only to students and faculty employed by the University of Tokyo**
+
+Office applications that are installed on your device, such as Word, Excel, and PowerPoint. You can use only the latest version of each application. For instructions on how to get started, please refer to the following.
+
+**[Installing Office Applications with UTokyo Microsoft License](./install)**{:.box.center}
+
+### OneDrive
+{:#onedrive}
+
+An online storage service provided by Microsoft. For instructions on how to use it, please refer to the following.
+
+**[OneDrive](./onedrive/)**{:.box.center}
+
+#### Microsoft Office Web Apps
+{:#office_web}
+
+To edit files on OneDrive in a web browser, you can use Microsoft Office Web Apps. Word, Excel, PowerPoint, etc. are available, but they do not have as many functions as the installed Office apps.
+
+Possible use cases include the following:
+- When minor editing of files on OneDrive is done entirely in the browser
+- When someone who is not a [Microsoft Office applications](#office_apps) user uses Office
+- When the Office app cannot be installed on the device due to permissions or other reasons
+
+For information on how to create and edit files on OneDrive using Microsoft Office Web Apps, please see **[Basic usage of OneDrive](./onedrive/basic)**.
+
+### Teams
+{:#teams}
+
+**Available only to faculty and staff**
+
+Teams is an application that is used for chats, video meetings and so on. For the instruction of usage, please refer to **[UTokyo Microsoft Teams User Manual (written in Japanese only)](https://univtokyo.sharepoint.com/sites/utokyoportal/wiki/SiteAssets/d/Useful_Tools/%E3%80%8C%E6%9D%B1%E4%BA%AC%E5%A4%A7%E5%AD%A6%E7%89%88_Microsoft_Teams%E5%88%A9%E7%94%A8%E3%83%9E%E3%83%8B%E3%83%A5%E3%82%A2%E3%83%AB%EF%BC%8820210311%EF%BC%89%E3%80%8D.pdf)** (accessible only to faculty and staff and written in only Japanese).
+
+#### "Team" Function
+{:#team_function}
+
+In November 2022, as part of our efforts to "utilize IT tools," in addition to the traditional "teams" based on the hierarchical structure of administrative organizations, it will become possible to use "teams" that can be composed of any members (application required). For manuals, please see below (accessible only to faculty and staff).
```

Comment:
```suggestion
In November 2022, as part of our efforts to "utilize IT tools," in addition to the traditional "teams" based on the hierarchical structure of administrative organizations, it will become possible to use "teams" that can be composed of any members (application required). For manuals, please see below (accessible only to faculty and staff and written in only Japanese).
```